File: app/src/main/java/com/quickseries/quickseriesapp/network/api/model/Categories.kt
package com.quickseries.quickseriesapp.network.api.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

/**
 *  Categories model
 *  @constructor Categories
 *
 */
@JsonIgnoreProperties(ignoreUnknown = true)
data class Categories(
    @field:JsonProperty("_id")
    var categoryId: String = "",
    var updated_at: String? = "",
    var slug: String? = "",
    var custom_module_eid: String? = "",
    var eid: String? = "",
    var title: String? = "",
    var description: String? = "",
    var __v: Int = 0,
    var _active: Boolean = false,
    var created_at: String? = ""
)

File: QuickSeries/QuickSeries/Model/Categories.swift
//
//  Categories.swift
//  QuickSeries
//
//  Created by Parth Patel on 13/05/20.
//  Copyright © 2020 Parth Patel. All rights reserved.
//

import Foundation

struct Categories : Codable {
    
    var id : String?
    var updatedAt : String?
    var slug : String?
    var customModuleEid : String?
    var eId : String?
    var title : String?
    var description : String?
    var v : Int?
    var active : Bool?
    var createdAt : String?
    
    enum CodingKeys: String, CodingKey {
        case id = "_id"
        case updatedAt = "updated_at"
        case slug
        case customModuleEid = "custom_module_eid"
        case eId = "eid"
        case title
        case description
        case v = "__v"
        case active = "_active"
        case createdAt = "created_at"
        
    }
    
}
